analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.cs

```diff
@@ -94,22 +94,67 @@ End Try
     Tag(""Else"")
 End If
 Tag(""End"")";
-            SETestContext.CreateVB(code).Validator.ValidateTagOrder(
-                "Entry",
-                "BeforeTry",
-                "Else",
-                "InTry",
-                "End",
-                "InCatch",
-                "InFinally",
-                "InFinally",    // With Exception
-                "AfterFinally");
-        }
+        var validator = SETestContext.CreateVB(code).Validator;
+        validator.ValidateTagOrder(
+            "Entry",
+            "BeforeTry",
+            "Else",
+            "InTry",
+            "End",
+            "InCatch",
+            "InFinally",
+            "InFinally",    // With Exception
+            "AfterFinally",
+            "End");
+        var symbol = validator.Symbol("Condition");
+        validator.TagStates("BeforeTry").Should().ContainSingle().Which[symbol].Should().HaveOnlyConstraints(BoolConstraint.True, ObjectConstraint.NotNull);
+        validator.TagStates("Else").Should().ContainSingle().Which[symbol].Should().HaveOnlyConstraints(BoolConstraint.False, ObjectConstraint.NotNull);
+        validator.TagStates("End").Should().SatisfyRespectively(
+            x => x[symbol].Should().HaveOnlyConstraints(BoolConstraint.False, ObjectConstraint.NotNull),
+            x => x[symbol].Should().HaveOnlyConstraints(BoolConstraint.True, ObjectConstraint.NotNull));
+    }
 
-        [TestMethod]
-        public void Branching_PersistSymbols_BetweenBlocks()
-        {
-            const string code = @"
+    [TestMethod]
+    public void Branching_BlockProcessingOrder_Parameter_VB()
+    {
+        const string code = @"
```

Comment:
What about using `"""`? It would allow you to replace all `""` below by `"` and you could also nicely indent to the level of the code:
```cs
"""
		Tag("Entry")
		If BoolParameter Then
		    Tag("BeforeTry")
		    Try
		        Tag("InTry")
		    Catch
		        Tag("InCatch")
		    Finally
		        Tag("InFinally")
		    End Try
		    Tag("AfterFinally")
		Else
		    Tag("Else")
		End If
		Tag("End")
		""");
```

Reply:
I have now changed the code so that this method is not touched. Thus I will not address this comment.